source/best-practices/backup.rst

```diff
@@ -1,61 +1,133 @@
 备份与还原
 ==========
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（作者）
+:最近更新日期: 2021-01-30
+:预计阅读时间: 20 分钟
+
+----
+
 **备份！备份！备份！**
 
 尽管不经常发生，但电脑随时可能会坏掉或被盗，电脑硬盘也随时可能会出问题，
-也可能不小心使用一个命令把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
-是十分危险的。
+也可能不小心使用一个命令（如 ``rm *``）把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
+是十分危险的。笔者和一些同行多次体验过数据误删或硬盘损坏又未备份带来的痛苦。
 
-备份的目的
+备份的策略
 ----------
 
+备份的策略简单概括有三种：
+
 - 仅备份重要文件
 - 仅备份家目录
 - 全盘备份
 
+不同备份策略的主要区别是所需的硬盘空间和备份时间。例如，若仅备份一些重要文件，
+则需要的硬盘空间较短，备份所需要的时间也较小。若要全盘备份，则需要较大的硬盘空间和
+较长的备份时间。仅备份家目录介于二者之间。
+
+建议用户根据个人需求和存储空间来决定具体的备份策略。如果某些文件非常重要，
+丢失后恢复难度极大或恢复时间极长，强烈推荐备份一下，如毕业论文、无法再次获取的数据、
+经过长时间处理得到的资料。如果拥有很大的硬盘存储空间，可以进一步选择备份家目录，
+甚至全盘备份。
```

Comment:
It would be better to list the pros and cons of each backup scheme. Maybe using a table?

Reply:
see d23dcc5.